api/src/main/java/net/kyori/adventure/text/renderer/TranslatableComponentRenderer.java

```diff
@@ -73,6 +73,13 @@
       protected @Nullable MessageFormat translate(final @NotNull String key, final @NotNull Locale context) {
         return source.translate(key, context);
       }
+
+      @Override
+      protected @NotNull Component renderTranslatable(final @NotNull TranslatableComponent component, final @NotNull Locale context) {
```

Comment:
I think it would be best to document the new behavior as well.

Reply:
👍🏻